types/lodash/common/object.d.ts

```diff
@@ -1022,15 +1022,17 @@ declare module "../index" {
 
     type GetIndexedField<T, K> = K extends keyof T
         ? T[K]
-        : K extends `${number}`
-            ? 'length' extends keyof T
-                ? number extends T['length']
-                    ? number extends keyof T
-                        ? T[number]
-                        : undefined
+        : K extends `${infer U extends number}`
+        ? "length" extends keyof T
```

Comment:
I'm having a hard time following this due to inconsistent indentation. I this this line and below should be indented 1 level deeper.

Reply:
Awesome, thanks for the quick feedback! I'll bump the `minimumTypescriptVersion` to `4.8` and write some unit tests for the change.

I'll also adjust the indentation level (for some reason prettier & eslint is acting up for me in the cloned repo).